packages/core/index.d.ts

```diff
@@ -204,3 +204,31 @@ export interface ZObject {
     delete: (key: string) => Promise<boolean>;
   };
 }
+
+export interface BufferedBundle<InputData = { [x: string]: any }> {
+  inputData: InputData;
+  meta: {
+    id: string;
+    [x: string]: any;
+  };
+}
+
+export interface PerformBulkItem {
+  outputData: { [x: string]: any };
+  error?: string;
+}
```

Comment:
To be consistent with #785, only _one_ of `outputData` or `error` is required. Maybe we could do this with two interfaces?

```typescript
export interface PerformBulkSuccessItem {
  outputData: { [x: string]: any };
}

export interface PreformBulkErrorItem {
  error: string;
}

export type PerformBulkItem = PerformBulkSuccessItem | PerformBulkErrorItem;
```

Reply:
I think these are the possible cases (`outputData` and `error`) or (`outputData`) or (`error`) for the and/or ;

```js
export interface PerformBulkSuccessItem {
  outputData: { [x: string]: any };
  error?: string;
}

export interface PreformBulkErrorItem {
  outputData?: { [x: string]: any };
  error: string;
}

export type PerformBulkItem = PerformBulkSuccessItem | PerformBulkErrorItem;
```

What do you think?

Reply:
So `outputData` and `error` can coexist. That makes sense!